ideas/homelab.md

```diff
@@ -0,0 +1,9 @@
+# Homelab
+
+|  Author  | Last Updated |       Time Estimate        |
+| :------: | :----------: | :------------------------: |
+| Jason Howe |  09/27/2022  | Short - 🟩🟩🟩🟩⬜️ - Long |
+
+## Description
+
+Do you have a homelab? Tell the club all about
```

Comment:
I think this project would be more about deploying a homelab than presenting about it. These are project ideas more than presentation ideas.